converter/src/main/java/de/ubleipzig/iiifproducer/converter/ConverterVersion3.java

```diff
@@ -0,0 +1,262 @@
+                    canvas.setId(canvasId);
+                    canvas.setType(CANVAS);
+                    canvas.setItems(annoPages);
+                    canvas.setHeight(height);
+                    canvas.setWidth(width);
+                    final String canvasLabel = c.getLabel();
+                    final Map<String, List<String>> canvasLabelMap = buildLabelMap(canvasLabel, NONE);
+                    canvas.setLabel(canvasLabelMap);
+                    canvases.add(canvas);
+                }
+            });
+
+            List<MetadataVersion3> finalMetadata = metadataBuilderVersion3.execute();
+            ManifestVersion3 newManifest = buildManifest(viewId, canvases);
+
+            //build structures objects
+            final Optional<List<Structure>> structures = ofNullable(manifest.getStructures());
+            if (structures.isPresent()) {
+                final List<Structure> structs = structures.get();
+                final StructureBuilderVersion3 sbuilder = StructureBuilderVersion3.builder()
+                        .structures(structs)
+                        .viewId(viewId)
+                        .build();
+                sbuilder.buildIncrements();
+                List<Item> newStructures = sbuilder.build();
+                newManifest.setStructures(newStructures);
+            }
+
+            //set seeAlso
+            final Optional<String> finalURN = ofNullable(getURNfromFinalMetadata(finalMetadata));
+            @SuppressWarnings("unchecked")
+            List<String> related = (List<String>) manifest.getRelated();
+
+            final List<SeeAlso> seeAlso = setSeeAlso(viewId, finalURN.orElse(null), related);
+            newManifest.setSeeAlso(seeAlso);
+            newManifest.setMetadata(finalMetadata);
+
+            //set manifest label
+            final String manifestLabel = manifest.getLabel();
+            final Map<String, List<String>> manifestLabelMap = buildLabelMap(manifestLabel, NONE);
+            newManifest.setLabel(manifestLabelMap);
+
+            return newManifest;
+        } catch (IOException ex) {
+            throw new RuntimeException("Could not Convert Manifest", ex.getCause());
+        }
+    }
+
+    public ManifestVersion3 buildManifest(final String viewId, final List<CanvasVersion3> canvases) {
+
+        final List<String> contexts = new ArrayList<>();
+        contexts.add(WEB_ANNOTATION_CONTEXT);
+        contexts.add(IIIF_VERSION3_CONTEXT);
+
+        final String id = baseUrl + viewId + separator + manifestBase + ".json";
+
+        final List<String> behaviors = new ArrayList<>();
+        behaviors.add("paged");
+
+        final Map<String, List<String>> label = buildLabelMap("Attribution", ENGLISH);
+        final Map<String, List<String>> value = buildLabelMap(domainAttribution, ENGLISH);
+        final MetadataVersion3 requiredStatement = MetadataVersion3.builder()
+                .label(label)
+                .value(value)
+                .build();
```

Comment:
Required statement can't be a single default statement. It should reproduce the content of v2 `attribution`. There are several projects at UBL that require different statements because they are still in copyright.

Reply:
see above new requiredStatement implementation